pkgs/development/hare-third-party/hare-ev/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, hare
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
the rec here isn't used so we should remove it.